cirq/contrib/quantum_volume/quantum_volume.py

```diff
@@ -474,11 +484,7 @@ def calculate_quantum_volume(
     )
 
     # Get the device graph from the given qubits or device.
-    device_graph = None
-    if isinstance(device_or_qubits, list):
-        device_graph = ccr.gridqubits_to_graph_device(device_or_qubits)
-    else:
-        device_graph = ccr.xmon_device_to_graph(device_or_qubits)
+    device_graph = ccr.gridqubits_to_graph_device(device_qubits)
```

Comment:
Copy-paste from #3894:

This is a breaking change. It's contrib, so I'm not deeply worried about breaking users, but having `deprecated_parameter` seems to imply backwards-compatibility that this doesn't provide.

My recommendation is to remove the deprecation warning, since we can't really make this backwards-compatible without a `cirq.google` dependency.

Reply:
sounds good!

Reply:
oh I realized that my deprecation can take care of this conversion path.